feat(changelog): cockpit-added-devtools-for-cockpit-! 2023-03-02 (#1379)
scaleway/docs-content: changelog/march2023/2023-03-02-cockpit-added-devtools-for-cockpit-!.mdx

---

> title: Devtools for Cockpit !

Reviewer: ```suggestion
title: Devtools for Cockpit are now in production!
```

status: added
author:

>     fullname: 'observability-beta'

Reviewer 2: ```suggestion
    fullname: 'Join the #observability-beta channel on Slack'
```

>     url: 'https://scaleway-community.slack.com/archives/C031S03BBF0'

Reviewer 2: ```suggestion
    url: 'https://scaleway-community.slack.com'
```

> date: 3/2/2023

Reviewer 2: ```suggestion
date: 2023-03-02
```

category: observability
product: cockpit
---

> Big news ! Devtools are now in production for Cockpit.

Reviewer: ```suggestion
Devtools are now in production for Observability Cockpit.
```

> You can use : 

Reviewer: ```suggestion
The following tools are available: 
```

> - SDK :

Reviewer: ```suggestion
- SDK:
```

  - Cockpit
  - Token
  - Grafana User
  - Alerts

> - CLI :

Reviewer: ```suggestion
- CLI:
```

  - Cockpit
  - Token
  - Grafana User

> - Terraform :

Reviewer: ```suggestion
- Terraform:
```
